.ci/targets/android_platform_tests.yaml

```diff
@@ -7,6 +7,10 @@ tasks:
   - name: lint
     script: script/tool_runner.sh
     args: ["lint-android"]
+  - name: download android deps
+    script: script/tool_runner.sh
+    infra_step: true
```

Comment:
Nice! I didn't realize they could be intermixed with other steps; this is much simpler that what I thought we'd have to do.